storage/src/checker/mod.rs

```diff
@@ -392,10 +446,14 @@ impl<S: WritableStorage> NodeStore<Committed, S> {
             }
             update_progress_bar(progress_bar, visited);
```

Comment:
this gets missed if we `continue` due to errors, which means the progress bar won't get updated when you get a bunch of errors.